cpp/src/robotick/framework/data/TelemetryServer.cpp

```
@@ -75,521 +79,237 @@ namespace robotick
-
-		if (!info)
+		if (!struct_desc || !workload_ptr)
 		{
-			res.body.set_from_string("{\"error\":\"not found\"}");
-			res.status_code = 404;
 			return;
 		}
 
-		WorkloadsBuffer& mirror = engine->get_workloads_buffer();
-		res.body.clear();
+		nlohmann::ordered_json struct_json;
```

Clang (14.0.6)

[warning] variable 'struct_json' is not initialized

(cppcoreguidelines-init-variables)